File: 프로그래머스/lv2/12985. 예상 대진표/예상 대진표.java
class Solution
{
    public int solution(int n, int a, int b)
    {
        int answer = 0;
        while (true) {
            answer++;
            int temp1 = (int) Math.ceil((double) a / 2);
            int temp2 = (int) Math.ceil((double) b / 2);
            if (temp1 == temp2) {
                break;
            } else {
                a = (int) Math.ceil((double) a / 2);
                b = (int) Math.ceil((double) b / 2);
            }
        }

        return answer;
    }
}

File: 프로그래머스/lv2/12985. 예상 대진표/예상 대진표.py
import math
def solution(n,a,b):
    answer = 0

    while True:
        answer += 1
        temp_a = math.ceil(a / 2)
        temp_b = math.ceil(b / 2)
        if temp_a == temp_b:
            break
        else:
            a = math.ceil(a / 2)
            b = math.ceil(b / 2)

    print(answer)
    return answer
